js/src/rive.ts

```diff
@@ -1452,6 +1456,65 @@ export class Rive {
     }
   }
 
+  public handleTouchEvents() {
+    if (!this.canvas || !this.animator.stateMachines.length || !this.renderer || !this.runtime) {
+      return;
+    }
+
+    const activeStateMachines =
+      this.animator.stateMachines.filter((sm: StateMachine) => sm.playing);
+    
+    function mouseCallback(event: MouseEvent) {
+      const boundingRect = (event.target as HTMLCanvasElement).getBoundingClientRect();
+
+      const canvasX = event.clientX - boundingRect.left;
+      const canvasY = event.clientY - boundingRect.top;
```

Comment:
Do we need to convert to canvas space when high DPI? i.e., multiply by window.devicePixelRatio?

Reply:
This calculation happens before the register touch event stuff gets called - where we use the window's dpi API to adjust the `canvas.width` and `canvas.height` so it should be taken into account by this point